src/DeviceInput/InputDevices/webDeviceInputSystem.ts

```diff
@@ -330,6 +330,15 @@ export class WebDeviceInputSystemImpl implements IDeviceInputSystem {
      * Handle all actions that come from pointer interaction
      */
     private _handlePointerActions() {
+        this._maxTouchPoints = navigator.maxTouchPoints || 0;

```

Comment:
Is it safe to use navigator without checking that it is available? Maybe to be safe just add a check `DomManagement.IsNavigatorAvailable()`.

```suggestion
        this._maxTouchPoints = (DomManagement.IsNavigatorAvailable() && navigator.maxTouchPoints) || 0;
```

Reply:
changed